src/lib/connect/ws.ts

```diff
@@ -263,13 +257,15 @@ const browserStreamBuilder: StreamBuilder = (client, opts) => {
 		if (socket.bufferedAmount > bufferSize) {
 			// throttle data until buffered amount is reduced.
 			setTimeout(socketWriteBrowser, bufferTimeout, chunk, enc, next)
+			return
```

Comment:
Wondering if there should be a return after setTimeout [here](https://github.com/mqttjs/MQTT.js/pull/1732/files#diff-cc6fb47ee82c471804f56a0629299e3e03381c08ce37d2b3053c259badf7fa64L265) ?

Seems it should: https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/bufferedAmount